src/tests/suite/daos_test_common.c

```diff
@@ -1350,3 +1346,24 @@ int wait_and_verify_pool_tgt_state(daos_handle_t poh, int tgtidx, int rank,
 
 	return -DER_TIMEDOUT;
 }
+
+/* TODO Link SPDK vmd.h to verify VMD LED enum states */
+int verify_led_state(int state, const char *state_str)
```

Comment:
This is not a common function, please move it to daos_nvme_recovery.c.

Reply:
there are quite a few non-common functions in daos_test_common.c, ie verify_led_state(), verify_state_in_log(), wait_and_verify_blobstore_state(). Maybe this is not the place for all these functions, but not sure they should go in daos_nvme_recovery.c either (since they may be used by other tests eventually?). Thoughts?

Reply:
yeah, TBH, I think some of them needs be moved out of daos_test_common.c. I'm find with cleanup them later.